src/data/markdown/docs/01 guides/05 Testing Guides/02 Load testing websites.md

```diff
@@ -3,115 +3,99 @@ title: "Load testing websites"
 
-Load testing in pre-production allows breaking your website without worrying about the interruption of your service.  Load testing in production is risky, but if your processes and team are mature, you could run your load tests in production as part of your <a href="https://principlesofchaos.org/">chaos experiments</a>. Sometimes it is the best way to get a picture of the real world.
+## How to load test a website?
 
-## Which type of performance tests should I run against my website?
+A load test will focus on testing the <b>performance of the requests to your backend</b>. The two most basic aspects to analyze are the <b>server response time</b> and the number of <b>returned errors</b>.
 
-The best results come when you run your tests in a systematic way, where you can compare data between test runs, and you change a minimal number of variables between tests. For this reason, we recommend the following testing pattern:
+For example, a load test simulating hundreds of concurrent users could validate that:
 
-### Baseline tests
+- Your server doesn't respond with errors.
+- The response time for 95% of your users should be below 400ms.
+- The response time of your images must always be below 600ms.
 
-A test with a small number of Virtual Users meant to produce response times for a system experiencing favorable conditions. You want a low enough number not to cause any stress on your system.
+Below, some common recommendations that you could consider when load testing a website:
 
-This test typically runs for 5-10 minutes, and it will establish the baseline for future comparisons.
-You could use the information on the analysis of these test results to create [thresholds](/using-k6/thresholds) in your future tests.
+### Decide what to test first
 
-### Stress tests
+ To begin, we recommend you to consider performance testing an iterative and continuous process. 
+ 
+ You start small, test, learn and iterate frequently.
 
-A test that steps through different levels of load and will highlight where performance problems start to happen.
-You should expect to iterate this test multiple times as you enter a test-> analyze-> make changes-> repeat pattern.
+> Start small and simple, make sure you get something out of the testing first, then expand the test suite and add more complexity until you feel that you’ve reached the point where more effort spent on realism will not give enough return on your invested time.
+> 
+> 
+> [Simple testing is better than no testing](https://k6.io/our-beliefs#simple-testing-is-better-than-no-testing)
 
+The first thing is to decide what to load test. On one side, you could evaluate which are the performance aspects that are most valuable to your business and has the most significant risks. On the other side, you could identify what your users do regularly. Use your analytics tools to find the most frequent user journeys.
```

Comment:
`On one side [...] on the other side` -> `On the one hand [...] on the other hand`,